index.ts

```diff
@@ -1,2 +1,46 @@
-import CashBalances from './data';
+import CashBalances, {CashBalanceReport, ReportSection, Account} from './data';
```

Comment:
```suggestion
import CashBalances, {ReportSection, Account} from './data';
```
Not used. Your linter should flag this if it's working